### PR TITLE
The Miner pipe retracting improvements

### PR DESCRIPTION
**Pull request by Relvl**
Added logic to start retracting the pipes when the miner are disabled.
Added lines to tooltip on Miner and Pump.

### CODE REVIEW

File: src/main/java/gregtech/common/tileentities/machines/basic/GT_MetaTileEntity_Miner.java [hunk 1]
```diff
@@ -217,45 +286,49 @@ public long maxEUStore() {
         return mTier == 1 ? 4096 : V[mTier] * 64;
     }
 
-    public boolean moveOneDown(IGregTechTileEntity aBaseMetaTileEntity) {
+    private void movePipeDown(IGregTechTileEntity aBaseMetaTileEntity) {
```
Comment (bombcar) [inline src/main/java/gregtech/common/tileentities/machines/basic/GT_MetaTileEntity_Miner.java:289]: In general we should keep *public* and instead add a new private and call that from the old public - who knows what is using basically anything

**Review by repo-alt — APPROVED** [review]
(After so many refactorings miner code still feels _old_)

**Comment by Relvl** [comment]
> (After so many refactorings miner code still feels _old_)

Well, actually most GT code feels old... Most of my refactoring are done to simplify code and reduce complexity. 
What specific changes are you calling for?

**Comment by repo-alt** [comment]
That was not about your changes, I just looked at the code that you were changing.

**Comment by Relvl** [comment]
> That was not about your changes, I just looked at the code that you were changing.

Well, i should not make a refactorings "just because" im sure. I would really like to improve many places in the mod, but edits for the sake of edits are not the best that can be. Also, if I commit too big changes, they will be harder to review.

**Pull request merged**